metpetdb_api/api/samples/v1/tests.py

```diff
@@ -193,7 +193,7 @@ def test_superuser_can_edit_an_unowned_sample(self):
         sample_data = deepcopy(self.sample_data)
         print "Sample_data:",sample_data
 
-        res = client.post('/samples/', sample_data)
+        res = client.post('/api/samples/', sample_data)
         print "Res:",res
         res_json = json.loads(res.content.decode('utf-8'))
 
```

Comment:
This line should be replaced with the following:

updated_sample_number = get_random_str()
sample_data['number'] = updated_sample_number